src/tiled/mainwindow.cpp

```diff
@@ -99,6 +102,95 @@ using namespace Tiled::Internal;
 using namespace Tiled::Utils;
 
 
+namespace {
+
+template <typename Format>
+struct ExportDetails
+{
+   Format* mFormat;
+   QString mFileName;
+
+   ExportDetails() {}
```

Comment:
Should initialize `mFormat` to `nullptr`.

Reply:
Done